teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './person'
+
+class Teacher < Person
+  attr_accessor @specialization
+
+  def initialize(specialization)
+    super
+    @specialization = specialization
+  end
```

Comment:
- [OPTIONAL] I would suggest updating the initialize as to below, the benefits of that is to not have confusion about what are you inheriting and also to give default values to them in case they do not have them passed 
```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```